CredentialPostTest/CredentialPostTest/Data/Models/ZwapstoreRequest.cs

```diff
@@ -0,0 +1,9 @@
+namespace CredentialPostTest.Data.Models
+{
+    public class ZwapstoreRequest
```

Comment:
Rename to `ClientRequest`?

Reply:
Maybe `public abstract class ZwapstoreRequestBase`? As I understand, this class was added in order to be used as a base class, right?